pandas/tests/formats/test_format.py

```diff
@@ -1125,6 +1125,17 @@ def test_to_string_no_header(self):
 
         self.assertEqual(df_s, expected)
 
+    def test_to_latex_specified_header(self):
```

Comment:
Sorry that was the wrong function name. It should be:
``test_to_string_specified_header``